src/main.cpp

```diff
@@ -3,14 +3,89 @@
 //  Game of Life
 //
 
+#include <SFML/Graphics.hpp>
 #include <cmath>
 #include <iostream>
+#include "Grid.hpp"
 
 
 namespace {
 
+const int SCREEN_WIDTH = 1420;
+const int SCREEN_HEIGHT = 1000;
+
 void run(char** argv) {
-    printf("Hello, world!\n");
+    
+    sf::Clock clock;
+    sf::Time elapsed;
```

Comment:
Don't use sf Clock and Time, use chrono facilities instead.

Reply:
okay, will look into this